backend/canisters/registry/api/src/updates/add_token.rs

```diff
@@ -2,11 +2,12 @@ use crate::TokenStandard;
 use candid::CandidType;
 use human_readable::{HumanReadablePrincipal, ToHumanReadable};
 use serde::{Deserialize, Serialize};
-use types::CanisterId;
+use types::{CanisterId, UserId};
 
 #[derive(CandidType, Serialize, Deserialize, Debug)]
 pub struct Args {
     pub ledger_canister_id: CanisterId,
+    pub submitted_by: Option<UserId>,
```

Comment:
When processing the token it gets rejected if this isn't set, so this should probably be a required field and we just get the website out first